src/pkg/caendr/caendr/utils/env.py

```diff
@@ -72,6 +72,44 @@ def get_env_var(key, value=None, can_be_none=False, as_template=False, var_type=
   return v
 
 
+
+def get_env_var_with_fallback(key, *fallback_keys, fallback_on_err=True, **kwargs):
```

Comment:
I appreciate the dynamic nature of this lookup, but worry that it introduces a level of implicit behaviors that might hide issues across different parts of the code base. Instead, if we need to revise the ENV vars, I would prefer we make it clearer at that level and set the protocol for *which* bucket should be used for various ends. Let's discuss.

Reply:
Hm, fair point. As far as the buckets go, I wonder if it would be better to replace generic bucket vars like `PUBLIC_BUCKET` and `PRIVATE_BUCKET` with specific ones like `DATASET_RELEASE_BUCKET` or `PHOTOS_BUCKET`.  We could maybe talk about setting defaults (e.g. `DATASET_RELEASE_BUCKET` falls back to `PUBLIC_BUCKET`), but to your point it would be better to define all the bucket variables explicitly.

Reply:
I really like this suggestion! Let's add this to the next cycle.